### ISSUE TITLE
Invalid ValueID passed to GetValueLabel

### ISSUE THREAD

**Issue by cburghardt** [OpenZWave/node-openzwave-shared#353]
Openzwave 1.6
Driver crashes as can be seen from the attached log.
Issue is described here:
https://github.com/OpenZWave/open-zwave/issues/1785

Apparently needs to be captured in the library.
[1571237396891-ozw_log3.txt](https://github.com/OpenZWave/node-openzwave-shared/files/3737851/1571237396891-ozw_log3.txt)

**Comment by robertsLando** [comment]
Needs a try catch statment so?

**Comment by cburghardt** [comment]
Yes but that is needed in your library so I don't understand your question

**Comment by robertsLando** [comment]
This is not my lib, I have asked to the author to give me write access to help him maintain this lib. I will check where that function is used in the code and add a try catch, if you want ypu can submit a pr

Daniel
